crates/rust-analyzer/src/config.rs

```diff
@@ -1743,6 +1756,22 @@ fn field_props(field: &str, ty: &str, doc: &[&str], default: &str) -> serde_json
                 { "type": "null" }
             ],
         },
+        "CargoUnsetTest" => set! {
+            "type": ["string", "array"],
+            "items": { "type": "string" },
+            "enum": ["all"],
+            "enumDescriptions": [
+                "All crates are analyzed without `--cfg=test`",
+            ],
+        },
+        "Option<CargoUnsetTest>" => set! {
+            "type": ["string", "array", "null"],
+            "items": { "type": "string" },
+            "enum": ["all"],
+            "enumDescriptions": [
+                "All crates are analyzed without `--cfg=test`",
+            ],
+        },
```

Comment:
This should be unnecessary

Reply:
Indeed. Not sure why I had to add it before.